app/presenters/any_link_presenter.rb

```diff
@@ -0,0 +1,92 @@
+class AnyLinkPresenter
```

Comment:
Would it be worth adding unit tests for this class? Also we still have the `LinkPresenter` and `LocalAuthorityPresenter` which appear to do similar things. Is there a possibility of getting rid of any of these classes?